components/Navigation.js

```diff
@@ -160,15 +160,15 @@ const Navigation = () => {
                 }
               </a>
             </Link> }
-            {TELEGRAM_URL && !isAuthenticated && <a
+            {/* {TELEGRAM_URL && !isAuthenticated && <a
```

Comment:
If we are removing this, we shouldn't keep the commented code. But why are we removing it?